.github/ISSUE_TEMPLATE/4-flaky-test.yml

```diff
@@ -0,0 +1,45 @@
+name: "❄️ Flaky internal test"
+title: "[Flaky internal test]: "
+labels: ["topic: flake ❄️", "stage: fire watch"]
+epics: ["Restore confidence in CI"]
```

Comment:
TBH I don't know if this `epics` line will work, but there's no way to test it without merging. If it doesn't work, only the template will be broken -- nothing else -- so it's safe to try out